src/modules/settings/components/settings/twitch/AutoPlay.jsx

```diff
@@ -2,40 +2,40 @@ import React from 'react';
 import Panel from 'rsuite/Panel';
 import Checkbox from 'rsuite/Checkbox';
 import CheckboxGroup from 'rsuite/CheckboxGroup';
-import {registerComponent, useStorageState} from '../Store.jsx';
-import {SettingIds, CategoryTypes, AutoPlayFlags} from '../../../../constants.js';
-import styles from '../../styles/header.module.css';
-import {hasFlag} from '../../../../utils/flags.js';
+import {registerComponent, useStorageState} from '../../Store.jsx';
+import {SettingIds, CategoryTypes, AutoPlayFlags} from '../../../../../constants.js';
+import styles from '../../../styles/header.module.css';
+import {hasFlag} from '../../../../../utils/flags.js';
 
 function AutoplayModule() {
   const [autoplay, setAutoplay] = useStorageState(SettingIds.AUTO_PLAY);
 
   return (
     <Panel header="Auto Play" className={styles.setting}>
       <div className={styles.setting}>
-        <p className={styles.description}>Control Auto Play on various Twitch video players.</p>
+        <p>Control Auto Play on various Twitch video players.</p>
         <CheckboxGroup
           value={Object.values(AutoPlayFlags).filter((value) => hasFlag(autoplay, value))}
           onChange={(value) => setAutoplay(value.length > 0 ? value.reduce((a, b) => a | b) : 0)}>
           <Checkbox key="fpPlayer" value={AutoPlayFlags.FP_VIDEO}>
             <p>Front-page Player</p>
-            <p className={styles.description}>Enable auto play for homepage video player</p>
+            <p>Enable auto play for homepage video player</p>
```

Comment:
why did we lose description css on all these? does the title not now match the description format?